var/spack/repos/builtin/packages/vtr/package.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from os.path import join as pjoin
+
+from spack.package import *
+
+
+class Vtr(MakefilePackage):
+    """The Verilog to Routing (VTR) project provides open-source CAD tools for
+    FPGA architecture and CAD research.
+    """
+
+    homepage = "https://verilogtorouting.org/"
+    git = "https://github.com/verilog-to-routing/vtr-verilog-to-routing.git"
+
+    maintainers("ueqri")  # For maintaining this package in the Spack community.
+
+    variant("eigen3", default=True, description="Build VTR with Eigen 3 library")
+
+    variant(
+        "tbb",
+        default=True,
+        description="Build VTR with Intel Threading Building Blocks (TBB) library",
+    )
+
+    variant(
+        "python",
+        default=True,
+        description="Install Python and required Python packages within Spack "
+        + "environment (not system wide)",
+    )
+
+    # Shallow clones on the 'master' branch will hinder VTR build.
+    version("master", branch="master", get_full_repo=True, preferred=True)
+    version("2023_NoC_Placement", tag="v2023_NoC_Placement")
+    version("8.0.0", tag="v8.0.0")
+
+    # C++14 compilers are required.
+    conflicts("%gcc@:6.1")
+    conflicts("%clang@:3.4")
+
+    depends_on("python@3.8:", type=("build", "run"), when="+python")
+    depends_on("py-pip", type=("build", "run"), when="+python")
+    depends_on("py-wheel", type=("build", "run"), when="+python")
+    depends_on("py-setuptools", type="build", when="+python")
```

Comment:
I would remove the variant above and simplify this as:
```suggestion
    depends_on("python@3.8:", type=("build", "run"))
    depends_on("py-pip", type=("build", "run"))
    depends_on("py-wheel", type=("build", "run"))
    depends_on("py-setuptools", type="build")
```

Reply:
Good idea!